### PR TITLE
051 bypalts

### PR DESCRIPTION
**Pull request by kgugala**
This PR enables fuzzer 051 to solve BYP_ALTs PIPs

### CODE REVIEW

**Comment by litghost** [comment]
Remove fuzzers/051-pip-imuxlout-bypalts/.generate.tcl.swp?

**Comment by litghost** [comment]
Need to update fuzzers/Makefile?

File: fuzzers/051-pip-imuxlout-bypalts/generate.tcl [hunk 1]
```diff
@@ -26,6 +26,49 @@ route_design
 
 # write_checkpoint -force design.dcp
 
+proc find_dst_pin {tile dst_wire} {
```
Comment (litghost) [inline fuzzers/051-pip-imuxlout-bypalts/generate.tcl]: Please add comments about what this does, inputs and outputs.  Same for find_src_pin.

File: fuzzers/051-pip-imuxlout-bypalts/generate.tcl [hunk 2]
```diff
@@ -62,8 +102,14 @@ for {set idx 0} {$idx < [llength $todo_lines]} {incr idx} {
...
     set mynet [create_net mynet_$idx]
     set mylut [create_cell -reference LUT1 mylut_$idx]
+    set dst_type [regsub {.*(.$)} $clb_dst_pin {\1}]
     set lutin [regsub {.*(.)} $clb_dst_pin {A\1}]
-    set_property -dict "LOC $slice BEL $lut LOCK_PINS I0:$lutin" $mylut
+
+    if { $dst_type >= 0 && $dst_type <= 6 } {
+        set_property -dict "LOC $slice BEL $lut LOCK_PINS I0:$lutin" $mylut
+    } else {
+        set_property -dict "LOC $slice BEL $lut" $mylut
```
Comment (litghost) [inline fuzzers/051-pip-imuxlout-bypalts/generate.tcl]: Can you add a comment on what this is doing?

**Comment by litghost** [comment]
Tests with PR look good, just need to address comments and good to merge.

File: fuzzers/051-pip-imuxlout-bypalts/generate.tcl (hunk 1 above)
Comment (kgugala) [reply fuzzers/051-pip-imuxlout-bypalts/generate.tcl]: done

File: fuzzers/051-pip-imuxlout-bypalts/generate.tcl (hunk 2 above)
Comment (kgugala) [reply fuzzers/051-pip-imuxlout-bypalts/generate.tcl]: done

**Comment by kgugala** [comment]
the swp file is now removed

**Comment by litghost** [comment]
Resolve the merge conflict and good to merge

**Comment by kgugala** [comment]
resolved

**Comment by litghost** [comment]
Actually, sorry.  fuzzers/Makefile still needs to be updated.

**Comment by kgugala** [comment]
yep, I forgot about this one. It is fixed now

**Pull request merged**